pkg/translator/loki/go.mod

```diff
@@ -4,7 +4,7 @@ go 1.18
 
 require (
 	github.com/go-logfmt/logfmt v0.5.1
-	github.com/grafana/loki v1.6.2-0.20221206154352-fee23745328c
```

Comment:
If you cannot remove `replace github.com/go-kit/log => github.com/dannykopping/go-kit-log v0.2.2-0.20221002180827-5591c1641b6b` this is useless change.

Reply:
this replace will be removed once i take out promtail in the following PR

Reply:
Just try that in the "pkg/translator/loki", that does not depend on promtail. If you cannot do that there you will not be able to do anywhere.